src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/Resources/Strings.resx

```diff
@@ -141,8 +141,12 @@
   <data name="LoggingMethodMustReturnVoidMessage" xml:space="preserve">
     <value>Logging methods must return void</value>
   </data>
+  <data name="MissingLoggerArgumentTitle" xml:space="preserve">
+    <value>One of the arguments to a static logging method must implement the Microsoft.Extensions.Logging.ILogger interface</value>
+    <comment>{Locked="Microsoft.Extensions.Logging.ILogger"}</comment>
+  </data>
   <data name="MissingLoggerArgumentMessage" xml:space="preserve">
-    <value>One of the arguments to a logging method must implement the Microsoft.Extensions.Logging.ILogger interface</value>
+    <value>One of the arguments to the static logging method `{0}` must implement the Microsoft.Extensions.Logging.ILogger interface</value>
```

Comment:
```suggestion
    <value>One of the arguments to the static logging method '{0}' must implement the Microsoft.Extensions.Logging.ILogger interface</value>
```
? I've never seen the usage of backticks here.
If I'm wrong, so please ignore that comment.

This applies to other places as well.

Reply:
Done. Thanks